libs/langgraph/langgraph/prebuilt/tool_node.py

```diff
@@ -214,6 +215,26 @@ def _func(
         config_list = get_config_list(config, len(tool_calls))
         with get_executor_for_config(config) as executor:
             outputs = [*executor.map(self._run_one, tool_calls, config_list)]
+
+        commands: list[Command] = [
+            output for output in outputs if isinstance(output, Command)
+        ]
+        # This can be relaxed by moving to a design where there is a single node per tool
+        # In that case multiple Commands can be handled natively by LangGraph
+        # (including concurrent state updates via reducers, multiple goto destinations, etc.)
+        if len(commands) > 1:
+            raise ValueError(
+                "Currently only one Command update per ToolNode is supported, got multiple Commands."
```

Comment:
Add better debug information? Which tools are the offending tools? What should a user do?